dask/dataframe/utils.py

```diff
@@ -869,7 +869,7 @@ def assert_sane_keynames(ddf):
         while isinstance(k, tuple):
             k = k[0]
         assert isinstance(k, (str, bytes))
-        assert len(k) < 100
+        assert len(k) < 120
```

Comment:
What's the invalid keyname you're getting?

Reply:
Yeah, this check is here to make sure that our keys don't get too long.  We should not relax it if possible.